src/hub-rest-client/openapi/.gitignore

```diff
@@ -0,0 +1,4 @@
+wwwroot/*.js
```

Comment:
Hm. You're right that it's unnecessary. I've added an `.openapi-generator-ignore` file to prevent creation of this an other unneeded noise.

Ignore-files ignoring other ignore-files!